akka-cluster-sharding/src/main/scala/akka/cluster/sharding/OldCoordinatorStateMigrationEventAdapter.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.cluster.sharding
+
+import akka.annotation.InternalApi
+import akka.cluster.sharding.ShardCoordinator.Internal.ShardHomeAllocated
+import akka.persistence.journal.EventAdapter
+import akka.persistence.journal.EventSeq
+
+/**
+ * Used for migrating from persistent state store mode to the new event sourced remember entities. No user API,
+ * used through configuration. See reference docs for details.
+ *
+ * INTERNAL API
+ */
+@InternalApi
+final class OldCoordinatorStateMigrationEventAdapter extends EventAdapter {
```

Comment:
private[akka] ?

Reply:
It's created through reflection, I have a feeling I had some problem when I privatised it before, but maybe I imagine that.

Reply:
it will be public in Java land so shouldn't be a problem